cli/connect/lua/eval_func_body.lua

```diff
@@ -1,6 +1,22 @@
 local yaml = require('yaml')
+yaml.cfg{ encode_use_tostring = true }
 local args = {...}
 local cmd = table.remove(args, 1)
+local is_sql_language = table.remove(args, 1)
+local need_metainfo = table.remove(args, 1)
+
+local function get_command(line)
+    if line:sub(1, 1) == '\\' then
+        return line:sub(2)
+    end
+    return nil
+end
+
+local command = get_command(cmd)
+if command or is_sql_language == true then
+    return require('console').eval(cmd)
+end
```

Comment:
It seems to me that this will only make things easier:
```suggestion
local function is_command(line)
    return line:sub(1, 1) == '\\'
end

if is_command(cmd) or is_sql_language == true then
    return require('console').eval(cmd)
end
```

Reply:
Updated.